plugin/core/sessions.py

```diff
@@ -265,12 +265,15 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
                 "insertTextModeSupport": {
                     "valueSet": [InsertTextMode.AdjustIndentation]
                 },
-                "labelDetailsSupport": True
+                "labelDetailsSupport": True,
             },
             "completionItemKind": {
                 "valueSet": completion_kinds
             },
-            "insertTextMode": InsertTextMode.AdjustIndentation
+            "insertTextMode": InsertTextMode.AdjustIndentation,
+            "completionList": {
+                "itemDefaults": ["editRange", "insertTextFormat", "data"]
```

Comment:
We don't support `commitCharacters` per individual completion but if we say that we do then that will potentially result in smaller payload right?

Reply:
The LSP client for ST didn't announce `commitCharactersSupport` in it's capabilities, 
so I would expect the server to not send the `commitCharacters`.

Reply:
Right. I did not recall that there is such capability. That's good news because it means I can drop those from typescript-language-server if the capability is missing.